shell/browser/api/electron_api_system_preferences_mac.mm

```diff
@@ -426,7 +426,7 @@ AVMediaType ParseMediaType(const std::string& media_type) {
 bool SystemPreferences::CanPromptTouchID() {
   base::scoped_nsobject<LAContext> context([[LAContext alloc] init]);
   if (![context
-          canEvaluatePolicy:LAPolicyDeviceOwnerAuthenticationWithBiometrics
+          canEvaluatePolicy:LAPolicyDeviceOwnerAuthenticationWithBiometricsOrWatch
```

Comment:
It looks like per [docs]() that this enumeration case is only available on macOS 10.15+, so we'll need to conditionally set that appropriately before this can be merged. 

That should look something like:

```
base::scoped_nsobject<LAContext> context([[LAContext alloc] init]);
LAPolicy auth_policy = LAPolicyDeviceOwnerAuthenticationWithBiometrics;
if (@available(macOS 10.15, *)) {
     auth_policy = LAPolicyDeviceOwnerAuthenticationWithBiometricsOrWatch;
}
if (![context canEvaluatePolicy:auth_ policy error:nil])
     return false;
 ... rest of function...
```

Reply:
indeed, nice catch. I just added that.